Automattic/wp-calypso
# composite-checkout: Replace TOS with CheckoutTerms

#### Changes proposed in this Pull Request

This PR replaces `WPTermsAndConditions` with the `CheckoutTerms` component from the regular checkout. 

Note that some of the styles are customized differently than the regular checkout; notably that the links are underlined.

**Before:**

![Screen Shot 2020-03-09 at 8 31 41 PM](https://user-images.githubusercontent.com/2036909/76268631-0ab32700-6245-11ea-8ec4-5b393df5fe05.png)

**After:**

![Screen Shot 2020-03-09 at 8 46 54 PM](https://user-images.githubusercontent.com/2036909/76269216-24556e00-6247-11ea-802a-2e72b19c08a4.png)

#### Testing instructions

Visit composite checkout with different combinations of products, particularly plan-only carts, domain carts, and multiple domain carts with different TLDs. Verify that the Terms of Service section looks the same as the non-composite checkout version.
